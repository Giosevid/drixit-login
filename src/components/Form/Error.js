import React from 'react'

const Error = ({ children }) => {
  return (
    <div style={{ color:'red', mnarginTop:'0.5rem' }}>
      {children}
    </div>
  )
}

export default Error