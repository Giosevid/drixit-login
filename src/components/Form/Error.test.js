import {render} from "@testing-library/react";
import React from "react";
import Error from "./Error";

describe('test component <Error/> ', () => {
  test('render without crashing', () => {
    const { asFragment } = render(<Error/>)
    expect(asFragment()).toMatchSnapshot();
  })
})
