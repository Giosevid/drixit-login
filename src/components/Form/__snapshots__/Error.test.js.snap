// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component <Error/>  render without crashing 1`] = `
<DocumentFragment>
  <div
    style="color: red;"
  />
</DocumentFragment>
`;
